stable/spark-operator/templates/webhook-cleanup-job.yaml

```diff
@@ -6,7 +6,6 @@ metadata:
   annotations:
     "helm.sh/hook": pre-delete
     "helm.sh/hook-delete-policy": hook-succeeded
```

Comment:
remove these instead

Reply:
These were in the previous chart too, I guess they kick in when the chart is uninstalled